ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -107,13 +285,14 @@ typeof define === 'function' && define.amd ? define(['exports'], factory) :
   };
 
   /**
-     * Converts decoded ibr structure data into three.js Line objects.
-     * @param {Object} structure structures decoded from raw ibr data
-     * @param {number} structureIndex overall index of the structure
+     * Converts decoded ibr structure data into three.js Line objects
+     and add them to scene.
+     * @param {Object} structure structures decoded from raw ibr data.
+     * @param {number} structureIndex overall index of the structure.
      * @return {Map.<String, List.<Object>>} objects Layer name and
-     corresponding list of three.js Line objects
+     corresponding list of three.js Line objects.
      */
-  function renderLayer(structure, structureIndex) {
+  function renderLayerAndSetToInvisible(structure, structureIndex) {
```

Comment:
not sure what and where the set to invisible applies here? also, the return value here doesn't really match the function name (creates mapping vs rendering)

Reply:
'SetToInvisible' is not necessary to be put on function name. I will add that to JSDoc instead.